streampark-console/streampark-console-webapp/src/views/flink/app/components/Dependency.vue

```diff
@@ -113,12 +116,15 @@
             }
           }
           if (invalidArtifact.length === 0) {
-            const id = groupId + '_' + artifactId;
+            const id = classifier != null ? groupId + '_' + artifactId + '_' + classifier : groupId + '_' + artifactId;
```

Comment:
I suggest add a new method for this

Reply:
Add new function 
function getId(groupId: string, artifactId: string, classifier: string) {
    return classifier != null ? groupId + '_' + artifactId + '_' + classifier : groupId + '_' + artifactId;
  }